### PR TITLE
finished code challenge

### PR DESCRIPTION
**Pull request by lhchandler4**
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

### CODE REVIEW

File: challenges/assets/README/LinkedList_README.md [hunk 1]
```diff
@@ -14,6 +14,13 @@ Insert has a big O of 1. Includes has a big O of n. Print also has a big O of n.
...
 
+4. append inserts a value to the end of the linked list
+
+5. insert before will insert a given value before another given value if it is in the linked list
+
+6. insert after will insert a given value after another given value if it is in the linked list
+
+
 ## Solution
 ![](../img/LinkedList.jpeg)
```
Comment (evanslaton) [inline challenges/assets/README/LinkedList_README.md:25]: This image isn't showing up!
